FreydCategoriesForCAP/gap/LinearClosure.gd

```diff
@@ -69,6 +69,32 @@ DeclareAttribute( "CoefficientsList",
 DeclareAttribute( "SupportMorphisms",
                   IsLinearClosureMorphism );
 
+####################################
+##
+#! @Section Functors
+##
+####################################
+
+#! @Description
+#!  The arguments are functor <A>F</A>$:C\to D$, some linear closure <A>linear_closure</A> of $C$ over some
+#!  commutative ring $S$ and a function <A>ring_map</A>; where $D$ is a linear category over some commutative ring $R$.
+#!  The <A>ring_map</A> is a function that converts an element $s$ in $S$ to an element in $R$,
+#!  such that $s$ and <A>ring_map</A>($s$) are mathematically equal but may live in different Gap rings.
```

Comment:
```suggestion
#!  such that <A>ring_map</A> defines a ring homomorphism.
```

Your construction works in the case when `ring_map`, from a mathematical point of view, is simply a ring homomorphism.